src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,33 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class IsoscelesTrapezoid extends Figure {
+    private int baseA;
+    private int baseB;
+    private int leg;
+    private double height;
+
+    public IsoscelesTrapezoid() {
+        super();
+        this.baseA = new Random().nextInt(100);
+        this.baseB = new Random().nextInt(100);
+        this.leg = new Random().nextInt(100);
```

Comment:
Random value generation should be handled in the `FigureSupplier` class, as per the checklist: 'You can pass random values to the constructor of a figure instead of generating them inside figure classes.'